k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -42,7 +46,15 @@ spec:
           mountPath: /dev
         - name: tmp
           mountPath: /tmp
-          mountPropagation: Bidirectional
+#          mountPropagation: Bidirectional
+        env:
+        - name: cstorid
+          value: "9368bb1c-5cfc-11e8-b6fb-54e1ad4a9dd4"
```

Comment:
Need to provide good explanation of this value. If we are keeping a live design doc etc, mention its link. However, this particular place needs comments explaining the rationale.
This looks like some magic puzzle numeric value to me.

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas